index.html

```diff
@@ -0,0 +1,229 @@
+---
+layout: default
+---
+<header class="py-5">
+  <div class="container">
+    <h1>{{ locale.site_title}}</h1>
+    <p class="lead">{{ locale.site_description }}</p>
+  </div>
+</header>
+
+{% for leap in site.leap_versions %}
+<section id="leap-{{leap.version | replace: '.', '' }}" class="py-5">
+  <div class="container">
+    <h2>
+      openSUSE Leap {{ leap.version }}
+      {% if leap.end_of_life %}
+      <span class="badge badge-danger">{{ locale.end_of_life }}</span>
+      {% endif %}
+    </h2>
+
+    <div class="row">
+      <div class="col-sm-6 col-md-4">
+        <div class="card mb-3">
+          <img class="card-img-top" src="/assets/images/cover-startup.svg">
+          <div class="card-body">
+            <h5 class="card-title">
+              {{ locale.startup_guide }}
+            </h5>
+            <a class="card-link"
+              href="https://doc.opensuse.org/documentation/leap/archive/{{ leap.version }}/startup/html/book.opensuse.startup/index.html">
```

Comment:
For now the configuration works nicely. If in future here are any changes, it is very easy to add more specific configuration in `_config.yml`.

Reply:
I don't agree -- The production website already has changes in its 15.2 URLs  which you're now overwriting (`-` instead of `.`). Please just use my suggestion (1.), it should be simple (if annoying) & it's a better solution than this.